content/en/certificates.md

```diff
@@ -3,12 +3,241 @@ title: Chain of Trust
 linkTitle: Chain of Trust (Root and Intermediate Certificates)
 slug: certificates
 top_graphic: 5
-lastmod: 2024-03-15
+lastmod: 2024-05-03
 show_lastmod: 1
 ---
 
+<div style="text-align: center">
 
-[![ISRG Certificate Hierarchy Diagram, as of December 2020](/images/isrg-hierarchy.png)](/images/isrg-hierarchy.png)
+_**Note:** This section describes Let's Encrypt's hierarchy as it will be on June 6, 2024._
```

Comment:
```suggestion
_**Note:** This section describes Let's Encrypt's hierarchy as it will be starting June 6, 2024._
```

Reply:
Also this should include a link saying: "To see a description of the hierarchy in use before June 6, 2024, \[see below\](...)"

Or, relatedly, if we wanted to future-proof the tenses (in case we neglect to update the page on June 6), this could say:

This section describes Let's Encrypt's hierarchy after June 6, 2024. For the hierarchy in use before June 6, 2024, \[see below\](...)

Reply:
Both good ideas. I've rephrased and added a link down to the old section.